cita-executor/core/src/cita_executive.rs

```diff
@@ -101,7 +101,7 @@ impl<'a, B: DB + 'static> CitaExecutive<'a, B> {
             //        got: U256,
             //    }
             // Need to change VMError defined in cita-vm.
-            return Err(ExecutionError::VM(VMError::NotEnoughBaseGas));
+            return Err(ExecutionError::NotEnoughBaseGas);
```

Comment:
remove the comment in last line.

Reply:
Why? This is a FIXME comment, and I also think the VMError should be redefined.